packages/lib/findDurationType.ts

```diff
@@ -0,0 +1,11 @@
+import { MINUTES_IN_DAY, MINUTES_IN_HOUR } from "./convertToNewDurationType";
+
+export default function findDurationType(value: number) {
+  if (value % MINUTES_IN_DAY === 0) {
+    return "days";
+  } else if (value % MINUTES_IN_HOUR === 0) {
+    return "hours";
+  } else {
+    return "minutes";
+  }
```

Comment:
No need for `if/else` if we're returning anyways
```suggestion
  if (value % MINUTES_IN_DAY === 0) return "days";
  if (value % MINUTES_IN_HOUR === 0) return "hours";
  return "minutes";
  }
```